frontend/static/css/index.css

```diff
@@ -135,4 +135,10 @@ input[type=number] {
   filter: drop-shadow(0.5px 0.5px 0.5px #000000);
 }
 
-.MuiButton-textInherit {color : '#111111';}
\ No newline at end of file
+.MuiButton-textInherit {color : '#111111';}
+
+@media (max-width: 929px) {
```

Comment:
Hey @ozoono , now that the app is also available on Umbrel some users are finding again this book misalignment on Safari for MacOS. Do you think removing this `929px` limit might extend this .css fix for Safari for desktop as well?

Reply:
Hi, @Reckless-Satoshi i think moving the first css block of this media-query:
 ```
 .MuiDataGrid-columnHeaders + div {
    width: auto !important;
  }
```
outside the media-query (so it will affect mobile and desktop), should work.
I am going to do a PR with this small fix

Reply:
I've manually checked this change on Safari and it works.